packages/sdks/javascript/src/api/generated/apis/users-environments-api.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable */
+/**
+ * devopness API
+ * Devopness API - Painless essential DevOps to everyone 
+ *
+ * The version of the OpenAPI document: latest
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { ApiBaseService } from "../../../services/ApiBaseService";
+import { ApiResponse } from "../../../common/ApiResponse";
+import { ArgumentNullException } from "../../../common/Exceptions";
+import { ApiError } from '../../generated/models';
+import { EnvironmentRelation } from '../../generated/models';
+
+/**
+ * UsersEnvironmentsApiService - Auto-generated
+ */
+export class UsersEnvironmentsApiService extends ApiBaseService {
```

Comment:
@brlipi Note that this `*Service` class file is new and won't be used in any service yet.

Take a look at the file `packages/sdks/javascript/src/services/UserService.ts` to see how these service classes are used. Look at the `passwords` property as a reference.

Reply:
`UsersEnvironmentsApiService` adicionado ao `UserService` no commit https://github.com/devopness/devopness/pull/770/commits/ea54d492e285c1fc1dafe1cd45a968eeff6ccae0